content/docs/fairing/fairing-overview.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Overview of Kubeflow Fairing"
+description = "Build, train, and deploy your ML training jobs remotely"
+weight = 5
++++
+
+Kubeflow Fairing streamlines the process of building, training, and deploying
+machine learning (ML) training jobs in a hybrid cloud environment. By using
+Kubeflow Fairing and adding a few lines of code, you can run your ML training
+job locally or in the cloud, directly from Python code or a Jupyter
+notebook. After your training job is complete, you can use Kubeflow Fairing to
+deploy your trained model as a prediction endpoint.
+
+## Quickstart
+
+Follow a quickstart guide to learn how to get started running training jobs
+remotely with Kubeflow Fairing:
+
+*  Learn how to [train and deploy a model on Google Cloud Platform (GCP) from
+   a local notebook][gcp-local].
+*  Learn how to [train and deploy a model on GCP from a notebook hosted on
+   Kubeflow][gcp-kubeflow].
+
+## What is Kubeflow Fairing?
+
+Kubeflow Fairing is a Python package that makes it easy to train and deploy ML
+models on [Kubeflow][kubeflow], [Kubernetes][kubernetes], or [Google AI
+Platform][ai-platform]. 
+
+Kubeflow Fairing packages your Jupyter notebook or Python function as a Docker
+image, then deploys and runs the training job on Kubeflow, Kubernetes, or AI
+Platform. After your training job is complete, you can use Kubeflow Fairing to
+deploy your trained model as a prediction endpoint on Kubeflow or Kubernetes. 
+
+The following are the goals of the [Kubeflow Fairing project][fairing-repo]:
+
+*  Easily run ML training jobs remotely: Streamline the process of running ML
+   training jobs in a hybrid cloud environment. 
+*  Write ML training jobs once, then train locally or in the cloud: Provide ML
+   practitioners with a way to write ML training jobs once, then run them
+   locally or in a hybrid cloud environment.
```

Comment:
I think both the points here are saying the same thing. Three things I feel we should highlight for Fairing are (rough sentences):
1. Easily package code along with dependency - users don't have to know about how to create a docker image and best practices around it
2. Abstract out underlying infra api so that users can work with high level api that is friendly to data scientists 
3. Performing end to end ml workflow like going from training to a deployed model

Reply:
I've updated the goals, PTAL.